materialize-sql-json/.snapshots/TestTableTemplate

```diff
@@ -1,10 +1,10 @@
 
 	CREATE TABLE one."reserved".checkpoints (
-			materialization TEXT,
-			key_begin BIGINT,
-			key_end BIGINT,
-			fence BIGINT,
-			checkpoint TEXT,
+			materialization TEXT NOT NULL,
```

Comment:
This snapshot change is not a result of this PR. It's from running re-running the sqlgen tests, which apparently don't ever get run in CI.